apple/macbook-air/4/default.nix

```diff
@@ -3,7 +3,7 @@
 {
   imports = [
     ../.
-    ../../../common/cpu/intel/sandy-bridge
+    ../../../common/gpu/intel/sandy-bridge
```

Comment:
Sounds good, we have to think about what to do with our flake.nix that also references this file: https://github.com/NixOS/nixos-hardware/blob/3980e7816c99d9e4da7a7b762e5b294055b73b2f/flake.nix#L267

in https://github.com/NixOS/nixos-hardware/pull/991 someone wanted to expand the list of exports, but I would actually suggest to deprecate the old modules so that we can make progress on: https://github.com/NixOS/nixos-hardware/issues/992

Reply:
I am in favour of removing the common exports. They can be used by the existing model exports.

If a system needs the common modules, perhaps that system can be upstreamed to nixos-hardware.

Removing these exports does not block usage, one can reference the source file directly if needed.

Reply:
Ok. Could you remove those exports that are affected with warning? I believe there at least some users out there.

Reply:
I added a trace directly to the flake.nix. That way modules within nixos-hardware do not trigger any warning or trace.